Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>you don't need to specify <code>out=None</code>  explicitly.</p><p>However, I'd actually use a logger (json or runtime) and plot the data coming out from them, so the tutorial has a nice plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yes, I'll add a plot of the energy</p>